Makefile

```diff
@@ -136,19 +136,19 @@ push-mmr-polkadot:
 
 containerize-ci-linux:
 	@docker build -f docker/ci-linux.dockerfile \
-		-t ${REPO}/ci-linux:2022-04-18  \
+		-t ${REPO}/ci-linux:2022-08-06  \
```

Comment:
Should the ci-linux version be extracted out into a variable?

Reply:
Yes, it will be more simple for an update.
I will do it in the next PR.